api/rosetta/data_integration_test.go

```diff
@@ -14,6 +12,8 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
+// +build integration
```

Comment:
I moved it because that way you can just _always_ ignore the licence header without risking that you miss the tags. You will get used to it. I actually was debugging for a short while because I missed the tags as it was before.